bin/fmt-external.php

```diff
@@ -850,14 +850,23 @@ protected function doWrite($message, $newline)
     protected function hasColorSupport()
     {
         if (DIRECTORY_SEPARATOR === '\\') {
-            return
-                '10.0.10586' === PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD
+            return (function_exists('sapi_windows_vt100_support')
```

Comment:
I don't know what caused the changes to this file. It was updated during the build step.